save_preferences/save_preferences.js

```diff
@@ -1,18 +1,19 @@
-var allRanges = document.querySelectorAll(".range-wrap");
```

Comment:
Where all those changes also made in the TS file?

Reply:
In the ts file it was already written as const. I think Olivers compile compiled it into var instead of const. I am using the compiler of intellij so I think that's the reason why it's changed